backend/lib/endpoints/users.js

```diff
@@ -6,39 +6,31 @@ const httpErrors = require("http-errors");
 async function routes(app) {
   const User = app.mongo.model("User");
 
-  app.get(
-    "/current",
-    { preValidation: [app.authenticate] },
-    async (req, reply) => {
-      // todo: get current user from JWT
-      const result = await User.findById("");
-      if (result === null) {
-        return reply.send(new httpErrors.NotFound());
-      }
-      return {
-        id: result._id,
-        firstName: result.firstName,
-        lastName: result.firstName,
-        email: result.email,
-      };
-    },
-  );
+  app.get("/current", { preValidation: [app.authenticate] }, async () => {
```

Comment:
Minor: might want to use the same formatting convention as [here](https://github.com/manualzuru/FightPandemics.com/pull/75/files#diff-dfa0a0cc233d1b2aef3fb1520d8b6507R8), to keep code style consistent.

Reply:
Or even adopt this style instead. It doesn't really matter for me, as long as it's consistent.

Reply:
Yeah actually we use `Prettier` to automatically format the code style, and I think this depends on the line length, so it's not really something we can really choose in these handlers I think...

Reply:
but maybe we can always put it on the next line...
I was actually looking to move the `{ preValidation } ` hook to the containing function so we can have a secured endpoint with all the subroutes, which is possible in other Node.js frameworks such as express with "nested Routers"... but I didn't find a way to do that yet with Fastify, so definitely open to suggestions!

Reply:
Yeah, sometimes Prettier and other code formatters don't always catch things like that. I personally prefer the `{ preValidation }` to be on a separate line, but again, I have no strong opinions about it.

Reply:
Alright well now it's consistent, we can still consider later to change it to multi-line as you prefer, I also don't have strong preferences about this